README.md

```diff
@@ -104,12 +104,21 @@ The add and remove parameters can come in the following forms:
 - `add file.as function Mainfunc`. This will inject after the definition of `Mainfunc` in `file.as`.
 - `add file.as function Mainfunc 15`. This will inject after the definition of `Mainfunc` in `file.as`, with an offset of 15 lines from the start of `Mainfunc`.
 
+There is also an additional target:
+```
+add file.as
+begin-content
+    // content to match. may be multiline.
+end-content + 1
```

Comment:
Maybe do something more generic than `file.as` and `+ 1`? e.g, `<filename>.as` and `+ <offset>`?